src/angular-lightweight-markdown-editor.js

```diff
@@ -196,7 +208,7 @@
                 text.substr(start + length)
             ].join("");
         },
-        startLinesWith: function(text, start, end, startWith) {
+        startLinesWith: function(text, start, end, startWith, iterator) {
```

Comment:
is there some reason for this parameter? If not please remove and I can merge this PR